libtcmu.c

```diff
@@ -704,7 +704,7 @@ int tcmulib_master_fd_ready(struct tcmulib_context *ctx)
 
```

Comment:
Is this fixing a bug you are hitting?

It seems in the cases you are returning points to structs you only moved the problem. For example tcmu_get_runner_handler will not crash now, but it returns a NULL pointer and I think every call point will just reference that.

For the places you return a numerical value like tcmu_get_dev_num_lbas we just have another weird bug where incorrect values are being used.

Is there any place below where we can possibly have a null device or handler, etc or in every case would it be a bug?

Reply:
> Is there any place below where we can possibly have a null device or handler, etc or in every case would it be a bug?

Nevermind. I see you added one in the next patch :)